UnityProject/Assets/Scripts/Objects/Research/Portal.cs

```diff
@@ -102,19 +109,26 @@ public override bool WillAffectObject(GameObject eventData)
 
 		public override void OnObjectEnter(GameObject eventData)
 		{
-			Teleport(eventData);
+			if (isOnCooldown) return;
+			_ = Teleport(eventData);
 		}
 
-		private void Teleport(GameObject eventData)
+		private async Task Teleport(GameObject eventData)
 		{
-			if(connectedPortal == null) return;
-
-			SparkUtil.TrySpark(gameObject, expose: false);
-
-			TransportUtility.TeleportToObject(eventData, connectedPortal.gameObject,
-				connectedPortal.ObjectPhysics.OfficialPosition, true, false);
+			await Task.Delay(200);
```

Comment:
didn't actually fix the issue reeee

Reply:
actually it did, I fixed the infinte loop that crashes the server; never promised to fix another unrelated issue.

Reply:
I will fix  it then